app/src/common/shared/com/igalia/wolvic/ui/widgets/NavigationBarWidget.java

```diff
@@ -823,7 +844,7 @@ private void exitResizeMode(ResizeAction aResizeAction) {
         }
         AnimationHelper.fadeOut(mBinding.navigationBarMenu.resizeModeContainer, 0, () -> onWidgetUpdate(mAttachedWindow));
         mWidgetManager.popBackHandler(mResizeBackHandler);
-        mTrayViewModel.setShouldBeVisible(!mAttachedWindow.isFullScreen() || !mAttachedWindow.isKioskMode());
+        mTrayViewModel.setShouldBeVisible(!mAttachedWindow.isFullScreen() && !mAttachedWindow.isKioskMode());
```

Comment:
Seems like a different bug. Mind putting it into another commit?

Reply:
Done: https://github.com/Igalia/wolvic/pull/645